### PR TITLE
Assignment1

### PR DESCRIPTION
**Pull request by A-Tanner**
Includes features request and two tests for feature request.

###  Note for the reviewer: ###

a11895d contains the code for the objectives of the assignment

c03e0ac contains the code for the extra credit of the assignment

f642cfe fixes a logic error in one test and removes redundant code from another

### CODE REVIEW

**Comment by sammyboy2000** [comment]
Writing a peer review.

**Comment by sammyboy2000** [comment]
# Assignment

- Pull Request title includes "Assignment 1" in the title ✔
- Pull Request targets Assignment 1 branch ✔
- Issue 1: Application no longer crashes ✔
- Issue 2: Unit test properly passes ✔

### Extra Credit
- Issue 3: Feature request implemented ✔
- Issue 3: Feature request unit tested ✔

Could not find any issues regarding the coding guidelines. There is excessive whitespace in the test program. It can easily be seen in the when comparing all changes to the Assignment1 branch. Great work!

**Review by mmwoodfo — COMMENTED** [review]
👍🏻

File: PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs [hunk 1]
```diff
@@ -73,6 +89,55 @@ public void GetFilePath_ReturnsFileThatExists()
         }
 
 
+
+        [TestMethod]
+        public void RandomizeArray_AltersQuestionArray()
+        {
+            bool isChanged = false;
```
Comment (GrantErickson) [inline PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs:96]: This should be called
IsChanged

**Pull request closed without merging**